tests/playwright-test/loader.spec.ts

```diff
@@ -612,3 +612,29 @@ test('should import export assignment from ts', async ({ runInlineTest }) => {
   expect(result.passed).toBe(1);
   expect(result.exitCode).toBe(0);
 });
+
+test('should support node imports', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
```

Comment:
You'll probably need the following two lines: https://github.com/microsoft/playwright/blob/main/tests/playwright-test/loader.spec.ts#L254-L255.

Reply:
Also, to engage esm loader, you need `playwright.config.ts: 'export default {}'`, otherwise ts files do not load as ES modules.

Reply:
thanks for the feedback! I previously had `playwright.config.ts: 'export default {}'` but when I removed it the tests still failed when I undid my fix, but I'll add in for explicitness.